core/services/standardcapabilities/delegate.go

```diff
@@ -82,7 +82,7 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 			return nil, errors.New("gateway connector is required for web API Trigger capability")
 		}
 		connector := d.gatewayConnectorWrapper.GetGatewayConnector()
-		triggerSrvc, err := webapi.NewTrigger(spec.StandardCapabilitiesSpec.Config, d.registry, connector, log)
+		triggerSrvc, err := webapitrigger.NewTrigger(spec.StandardCapabilitiesSpec.Config, d.registry, connector, log)
```

Comment:
No need to rename the import here, it reads as `webapi.NewTrigger` unmodified which is clear about what it is.

Reply:
It's actually coming in as trigger but ok.